styles/index.css

```
@@ -40,178 +21,203 @@
         -ms-flex-align: center;
             align-items: center;
     background-color: #cad3d3;
+    text-align: center;
+    max-width: 100%;
 }
 
-.item {
-    width: 362px;
-    height: 768px;  /* nên dùng height: auto */
+.pricing-table, .pricing-table--featured {
```

nếu là modifier thì nên viết như này em nhé, như vậy độ ưu tiên của `.pricing-table--featured` sẽ cao hơn
```suggestion
.pricing-table, .pricing-table.pricing-table--featured {
```